ga4gh/backend.py

```diff
@@ -192,6 +198,68 @@ def _getEnd(cls, variant):
         return variant.end
 
 
+class VariantAnnotationsIntervalIterator(IntervalIterator):
+    """
+    An interval iterator for annotations
+    """
+
+    def __init__(self, request, parentContainer):
+        super(VariantAnnotationsIntervalIterator, self).__init__(
+            request, parentContainer)
+        print('self._request.featureIds: ', self._request.featureIds)
+        if self._request.featureIds is None:
+            self._featureIds = []
+        else:
+            self._featureIds = set(self._request.featureIds)
+        self._effects = set(self._request.effects)
```

Comment:
When the effects aren't in the request it has the same problem as null featureIds.

     --------------------------------------------------------------------------------
     ERROR in app [/Users/david/virtualenvs/ga4gh-server-1/lib/python2.7/site-packages/flask/app.py:1423]:
     Exception on /variantannotations/search [POST]
--------------------------------------------------------------------------------
...
  File "/Users/david/ga4gh/server-pablo2/ga4gh/backend.py", line 214, in __init__
    self._effects = set(self._request.effects)
     TypeError: 'NoneType' object is not iterable
     127.0.0.1 - - [19/Dec/2015 10:30:25] "POST /variantannotations/search HTTP/1.1" 500 -